includes/core.php

```diff
@@ -80,3 +84,73 @@ function maybe_disable_upload_image( $attachment_id ) {
 
 	return $attachment_id;
 }
+
+/**
+ * Adds tweeted status columns for supported post types.
+ */
+function add_tweeted_column() {
+	// Loop through all the supported post types and add tweet status column.
+	$post_types = Utils\get_enabled_post_types();
+	foreach ( (array) $post_types as $post_type ) {
+		add_post_type_support( $post_type, POST_TYPE_SUPPORT_FEATURE );
+		add_filter( "manage_{$post_type}_posts_columns", __NAMESPACE__ . '\modify_post_type_add_tweet_status_column' );
+		add_action( 'manage_' . $post_type . '_posts_custom_column', __NAMESPACE__ . '\modify_post_type_add_tweet_status', 10, 2 );
+	}
+}
+
+/**
+ * Add 'Tweeted' column for supported post types.
+ *
+ * @param array $columns Supported columns for a post type.
+ */
+function modify_post_type_add_tweet_status_column( $columns ) {
+	// Do this so our custom column doesn't end up being the last one, messing up UI.
+	unset( $columns['date'] );
+
+	// Add tweet status column header.
+	$columns['is_tweeted'] = sprintf(
+		'<span class="autoshare-for-twitter-status-logo" title="%s"><span class="screen-reader-text">%s</span></span>',
+		esc_attr__( 'Tweeted status', 'autoshare-for-twitter' ),
+		esc_html__( 'Tweeted status', 'autoshare-for-twitter' )
+	);
+
+	// Add the date column back.
+	$columns['date'] = esc_html__( 'Date', 'autoshare-for-twitter' );
+
+	return $columns;
+}
+
+/**
+ * Add tweet status data to each row.
+ *
+ * @param  string $column_name Column name.
+ * @param  int    $post_id Post ID.
+ */
+function modify_post_type_add_tweet_status( $column_name, $post_id ) {
+	if ( 'is_tweeted' === $column_name ) {
+		$post_status = get_post_status( $post_id );
+
+		$tweet_status = Utils\get_autoshare_for_twitter_meta( $post_id, TWITTER_STATUS_KEY );
+		$status       = isset( $tweet_status['status'] ) ? $tweet_status['status'] : '';
+
+		if ( 'publish' !== $post_status || empty( $status ) ) {
+			echo '';
+		} else {
+			if ( 'published' === $status ) {
+				$date        = Utils\date_from_twitter( $tweet_status['created_at'] );
+				$twitter_url = Utils\link_from_twitter( $tweet_status['twitter_id'] );
+				$tweet_title = sprintf(
+					'%s %s',
+					__( 'Tweeted on', 'autoshare-for-twitter' ),
+					$date
+				);
+
+				printf(
+					'<a href="' . esc_url( $twitter_url ) . '" target="_blank">
+					<span class="autoshare-for-twitter-status-logo" title="' . esc_attr( $tweet_title ) . '"></span>
```

Comment:
`title` attribute should be added to `<a>` instead.

Reply:
@dinhtungdu updated, please check again.